index.html

```diff
@@ -12,6 +12,122 @@
       <a class="menu"> Develover Tools </a>
     </div>
     <div class="page-content">
+      <div class="dev">
+        <div class="dev-header">
+          <img class="dev-header-triangles" src="images/triangle.svg">
+          <div class="dev-header-title">
+            Front-End Tools
+          </div>
+        </div>
+        <div class="dev-content">
+          <div class="tech-card">
+            <div class="tech-card-background background-clear"></div>
+            <div class="tech-card-background background-filled"></div>
+            <div class="tech-card-content">
+                <div class="tech-card-content-image-container">
```

Comment:
No need for this `div`. Avoid wrapping only one element whenever possible :smile: